Fix orphaned package libfile-find-rule-perl

**What happened:**
As of now, there is a bug with the [libfile-find-rule-perl](https://packages.debian.org/bookworm/libfile-find-rule-perl) package which let's the chroot test `test_orphaned` fail because this package is considered to be an orphaned package.

This morning, a new version of `debootstrap` has been released in Debian testing (See [here](https://tracker.debian.org/news/1366179/debootstrap-10127nmu1-migrated-to-testing/)) which seems to have an issue in its dependency handling.

As the [changelog](https://metadata.ftp-master.debian.org/changelogs//main/d/debootstrap/debootstrap_1.0.127+nmu1_changelog) stats, there has been a change with the `usrmerge` package. In the newest version, it is excluded from the package list which is installed during the bootstrap process. However, it's dependencies are still installed leaving them behind as orphaned ones.

**Debug Logs:**
```
=========================== short test summary info ============================
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[rlogin]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[rsh]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[rcp]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[telnet]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[libdb5.3:amd64]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[libdb5.3:arm64]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[libssl1.1:amd64]
PASSED base/test/test_blacklisted_packages.py::test_blacklisted_packages[libssl1.1:arm64]
PASSED base/test/test_duplicate_uids.py::test_duplicate_uids
PASSED base/test/test_groups.py::test_groups[root-user0]
PASSED base/test/test_groups.py::test_groups[wheel-user1]
PASSED base/test/test_kernel_config.py::test_kernel_config[args0]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[features-config0-base,kvm,_dev-cloud,kvm,_slim,base,_dev]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[features-config1-base,kvm,_dev-kvm,_slim,base,_dev]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[features-config2-base,kvm,_dev-kvm,_slim,base,_dev]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[cname-config3-kvm,_dev,chost-kvm_dev]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[flags-config4-base,kvm,_dev-_slim,_dev,firewall]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[elements-config5-base,kvm-cloud,kvm,base]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[ignore-config6-base,kvm-garden-feat: warning: No feature is ignored.]
PASSED base/test/test_local_garden_feat.py::test_garden_feat[params-config7-base,kvm-raw]
PASSED base/test/test_password_hashes.py::test_password_hashes
PASSED base/test/test_password_shadow.py::test_password_shadow
PASSED base/test/test_proc.py::test_proc
PASSED base/test/test_rkhunter.py::test_rkhunter
PASSED base/test/test_sgid_suid_files.py::test_sgid_suid_files[sgid-whitelist_files0]
PASSED base/test/test_sgid_suid_files.py::test_sgid_suid_files[suid-whitelist_files1]
PASSED base/test/test_tiger.py::test_tiger
PASSED base/test/test_umask.py::test_umask[/etc/login.defs-dict0]
PASSED base/test/test_users.py::test_users
PASSED cloud/test/test_autologout.py::test_autologout[/etc/profile.d/50-autologout.sh-dict0]
PASSED cloud/test/test_packages_musthave.py::test_packages_musthave
PASSED cloud/test/test_pam.py::test_pam_faillock[/etc/pam.d/common-password*** pam_passwdqc.so min=disabled,disabled,12,8,8 passphrase=4 similar=deny retry=5]
PASSED cloud/test/test_pam.py::test_pam_faillock[/etc/pam.d/common-password*** pam_pwhistory.so use_authtok remember=5 retry=5]
PASSED cloud/test/test_pam_faillock.py::test_pam_faillock[/etc/pam.d/common-auth-auth required pam_faillock.so preauth silent audit deny=5 unlock_time=900]
PASSED cloud/test/test_pam_faillock.py::test_pam_faillock[/etc/pam.d/common-auth-auth [default=die] pam_faillock.so authfail silent audit deny=5 unlock_time=900]
PASSED cloud/test/test_pam_faillock.py::test_pam_faillock[/etc/pam.d/common-account-account required pam_faillock.so]
PASSED kvm/test/test_packages_musthave.py::test_packages_musthave
PASSED server/test/test_autologin.py::test_autologin[/etc/systemd/system/serial-getty@.service.d/autologin.conf-args0]
PASSED server/test/test_autologin.py::test_autologin[/etc/systemd/system/getty@tty1.service.d/autologin.conf-args1]
PASSED server/test/test_capabilities.py::test_capabilities
PASSED server/test/test_history.py::test_history[/etc/profile.d/50-nohistory.sh-args0]
PASSED server/test/test_kernel_parameter.py::test_kernel_parameter[fs.protected_symlinks-1]
SKIPPED [1] ../features/gcp/test/test_users.py: test is not part of the enabled features
SKIPPED [1] ../features/khost/test/test_systemd_unit.py:6: test is not part of the enabled features
SKIPPED [4] ../features/openstack/test/test_check_files.py:5: test is not part of the enabled features
SKIPPED [3] ../features/openstack/test/test_content_in_file.py:6: test is not part of the enabled features
SKIPPED [2] ../features/vhost/test/test_sgid_suid_files.py:6: test is not part of the enabled features
SKIPPED [1] ../features/vhost/test/test_systemd_unit.py:6: test is not part of the enabled features
SKIPPED [1] ../features/vhost/test/test_users.py:4: test is not part of the enabled features
SKIPPED [6] ../features/vmware/test/test_check_files.py:5: test is not part of the enabled features
SKIPPED [3] ../features/vmware/test/test_content_in_file.py:6: test is not part of the enabled features
SKIPPED [4] conftest.py:400: test only supported on azure
SKIPPED [1] conftest.py:390: test only supported on ali
SKIPPED [3] integration/test_gardenlinux.py:172: ipv6 not available in all vpcs
SKIPPED [1] conftest.py:430: test only supported on kvm
SKIPPED [1] conftest.py:450: test only supported on openstack
SKIPPED [3] conftest.py:410: test only supported on aws
SKIPPED [2] conftest.py:420: test only supported on gcp
FAILED base/test/test_orphaned.py::test_orphaned - AssertionError: Found orph...
======= 1 failed, 68 passed, 101 skipped, 3 warnings in 89.56s (0:01:29) =======
```

**How to reproduce it**:
```
make kvm-dev
```

**Environment:**

| Name | Value |
| ------------------- | ----- |
| Garden Linux Branch | main |
| Container runtime (version)  | podman (3.0.0) | 
| Host OS of build machine  | Debian 11 |

--- a/features/base/exec.config
+++ b/features/base/exec.config
@@ -13,3 +13,15 @@ chmod u-s /bin/umount /bin/mount
 # Mark package as manual installed to pass the orphaned test
 # The package is installed and required by debootstrap 1.0.127
 apt-mark manual usr-is-merged
+
+# Issue 1305
+# Mark some packages as manual installed so that the orphaned test
+# is not complaining. These packages are installed due to a bug
+# in debootstrap 1.0.127+nmu1 which installs the dependencies of
+# 'usrmerge' but leaves the package itself behind.
+apt-mark manual \
+  libfile-find-rule-perl \
+  libgdbm-compat4 \
+  libnumber-compare-perl \
+  libperl5.34 \
+  libtext-glob-perl

